databases/week1/week1-homework.sql

```diff
@@ -0,0 +1,48 @@
+-- 1- Find out how many tasks are in the task table.
+select count(*) as Total_number_of_task from task;
+-- 2- Find out how many tasks in the task table do not have a valid due date.
+select sum(case when due_date IS NULL then 1 else 0 end) as NUMBER_OF_NULL_VALUE from task;
+select * from task where due_date is null;
+-- 3- Find all the tasks that are marked as done.
```

Comment:
This query is a bit more simple than what you have done here, the users shouldn't be involved and the group by is not needed. Can you try again please?

Also let me know if the exercise is not clear :)